source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -237,7 +244,7 @@ void AuthenticatorImpl::onDestroy() {
 
 // Verify with a specific public key.
 void AuthenticatorImpl::verifyKey() {
-  const Status status = ::google::jwt_verify::verifyJwt(*jwt_, *jwks_data_->getJwksObj());
+  const Status status = ::google::jwt_verify::verifyJwtWithoutTimeChecking(*jwt_, *jwks_data_->getJwksObj());
```

Comment:
You need to get the latest jwt_verify_lib [here](https://github.com/envoyproxy/envoy/blob/master/bazel/repository_locations.bzl#L189):

By change its SHA, and sha256. 

the sha256 sum is by: downloading that .tar.gz file, and run sha256sum on the file.

Reply:
How do I get the current .tar.gz file. As I understand, the file
https://github.com/google/jwt_verify_lib/archive/40e2cc938f4bcd059a97dc6c73f59ecfa5a71bac.tar.gz

is outdated. But how do I get the new one?

Reply:
the new sha is: b5b3b4ed8611b1eea8764845381e60becc7b0b43

the file is  https://github.com/google/jwt_verify_lib/archive/b5b3b4ed8611b1eea8764845381e60becc7b0b43.tar.gz

sha256 is:  21b9fd9fb8714cb199a823a4c01cf6665bdd42b62137348707dee51714797dfc

Reply:
please also update the date in the comment field